app/models/SuperModel.coffee

```diff
@@ -132,9 +140,13 @@ module.exports = class SuperModel extends Backbone.Model
   # Tracking resources being loaded for this supermodel
 
   finished: ->
-    return @progress is 1.0 or not @denom
+    return (@progress is 1.0) or (not @denom) or @failed 
```

Comment:
We never unset `@failed`–is it intentional that the SuperModel will permanently be in a finished state after any resources fail? I know we don't retry any more, but won't this make it so that if you click some other link where the SuperModel is preserved across links, then the new page will also not work?